nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/trash/TrashService.java

```diff
@@ -132,6 +132,15 @@
      */
     DocumentModelList getDocuments(DocumentModel currentDoc);
 
+    /**
+     * Get all document references from the trash of the current document.
+     *
+     * @since 10.1
+     * @param currentDoc The current/parent document of trash document.
```

Comment:
For parameters and return, no initial capital and no terminating colon.

Could you rephrase the Javadoc and param description? I don't understand. What's "the trash of the current document"? The method name is not clear either imho.

Reply:
:+1: . FYI I just copied adn adapted the org.nuxeo.ecm.core.trash.TrashService.getDocuments(DocumentModel) doc but I can indeed improve it.